Extensions/ValuesOfMultipleObjects.json

```diff
@@ -0,0 +1,1796 @@
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "objects",
+          "longDescription": "",
+          "name": "objects",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Maximum Y position of picked object instances (using AABB of objects)",
+      "fullName": "Maximum Y position of picked object instances (using AABB of objects)",
+      "functionType": "Expression",
+      "name": "MaxY_AABB",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "// Get the first instance picked\nconst object = objects[0];\n\n// If the instance doesn't exist we exit this function, returnValue will be 0\nif (!object) return;\n\n// Get the AABB of the object\nconst aabb = object.getAABB();\n\n// Get the layer of the current object\nconst layer = runtimeScene.getLayer(object.getLayer());\n\neventsFunctionContext.returnValue = aabb.max[1];\n",
+          "parameterObjects": "objects",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "objects",
+          "longDescription": "",
+          "name": "objects",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Minimum X position of picked object instances (using center of objects)",
+      "fullName": "Minimum X position of picked object instances (using center of objects)",
+      "functionType": "Expression",
+      "name": "MinX_Center",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Comment",
+          "color": {
+            "b": 109,
+            "g": 230,
+            "r": 255,
+            "textB": 0,
+            "textG": 0,
+            "textR": 0
+          },
+          "comment": "Initialize with a value from an object (to prevent 0 from being used)",
+          "comment2": ""
+        },
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "AjoutHasard"
+              },
+              "parameters": [
+                "",
+                "Object"
+              ],
+              "subInstructions": []
+            }
```

Comment:
```suggestion
```

I'd remove this "Pick a random object" condition since you do not care that if object is randomly picked or not. Removing it should always take the one from the first object in the picked objects list, which is fine here. Picking a random one just adds overhead. Same applies to other functions that use that condition.

Reply:
Great optimization!  However, it is no longer needed as those functions have been removed.